docs/learn/betting_strategy.md

```diff
@@ -0,0 +1,269 @@
+---
+id: betting-strategy
+title: Betting Strategies
+---
+
+# Betting Strategies
+
+What follows is a short introduction to the basic strategies when dealing with
+prediction markets. It is by no means a complete guide.
+
+Recall that the asset prices in prediction markets as used to form a prediction
+of future events, and the right AMMs are _proper_ in the sense that they
+incentivize the traders (which are in this context often referred to as
+_informants_) to change the prices so that they reflect their beliefs.
+
+Before you become an informant and expose yourself to a prediction market, you
+need a _belief_. This belief could be anything from "$A$ is definitely going to
+occur" or "$A$ is _not_ going to occur with a probability of 80%" to "The
+following probability distribution is correct", etc.
```

Comment:
I have to highlight that this is for me the most interesting part of prediction markets, because prediction markets allow you to reconsider your decisions looking at probabilities made by market rules.